### ISSUE TITLE
renaming ext_grid?

### ISSUE THREAD

**Issue by lthurner** [e2nIEE/pandapipes#50]
As far as I can tell, ext_grid ist the only element that is named the same in pandapipes and pandapower. All other names seem to be unique (bus/junction, line/pipe etc.). Is that correct? And if so, wouldn't it make sense to rename ext_grid in pandapipes to something else, so that all names are unique?

Seems to me this would make it easier to write applications that combine pandapower and pandapipes.

**Comment by dlohmeier** [comment]
Probably you are right about the combining applications. We had this in mind for a long time, but it is hard to come up with a reasonable name for this component. I think it should somehow indicate that a value (p, T) is fixed. So either we split up two components and name them fixed_p_node and fixed_T_node or something the like, or we keep them combined and use fixed_variable_node. Or are there other suggestions?

**Comment by SimonRubenDrauz** [comment]
After a long discussion in the pandapipes-Team we decided not to rename the external grid. We just did not come up with a solution which is as clear as external grid, does not lead to confusion and still is in coherence we the pandapower structure. Reasons:
- As in pandapower the ext_grid does something similar as the 'ext_grid' in pandapipes, we think, it should have at least have 'ext_grid' in the name.
- Splitting p and T is not a nice solution as in the end you would come up with at least two different kinds of ext_grids: ext_grid_p, ext_grid_T. We think, this might cause more confusion, then it makes it easier, as in a heat grid you always need both. Offering a ext_grid_p, ext_grid_pT and ext_grid_T, in our eyes, even increases the confusion and might lead to not well-defined behavior, as you also need to cover cases where you combine all three of them making the programming solution much more complicated.
- ext_grid_fluid as another solution seems also not appropriate in our eyes, as someone might think, if there is a ext_grid_fluid, there might be additional ext_grids (like ext_grid_gas, ext_grid_heat, ...).

Moreover, so far all our implementations regarding mutli_energy_grids comprise clear solutions which are able to handle that in pandapower and pandapipes nets external grids are called the same. Therefore, we think, this shouldn't be a problem when combining pandapower and pandapipes.

Therefore, we still think it is the best to stick to the name ext_grid despite the fact, that all other components except for the ext_grid are named differently. However, if you think of a better solution @lthurner, we still are open for changes. If you are okay with it, that we stick to the name, I would close the issue.
